modules/multi-architecture-modify-machine-set-aws.adoc

```diff
@@ -0,0 +1,120 @@
+//Module included in the following assembly
+//
+//post_installation_configuration/cluster-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="multi-architecture-modify-machine-set-aws_{context}"]
+
+= Adding an ARM64 compute machine set to your cluster 
+
+To configure a cluster with multi-architecture compute machines, you must create a AWS ARM64 compute machine set. This adds ARM64 compute nodes to your cluster and creates a cluster with multi-architecture compute machines.
+
+.Prerequisites 
+
+* You installed the OpenShift CLI (`oc`). 
+* You created an AMD64 single-architecture AWS installer-provisioned cluster with the multi-architecture installer binary.
+
+
+.Procedure
+
+* Create and modify a compute machine set, this will control the ARM64 compute nodes in your cluster. 
++ 
+[source,terminal]
+----
+$ oc create -f aws-arm64-machine-set-0.yaml
+----
+
++
+.Sample YAML compute machine set to deploy a ARM64 compute node
+
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  labels:
+    machine.openshift.io/cluster-api-cluster: <infrastructure_id> 
```

Comment:
Usually machine set topics explain where to get the `<infrastructure_id>`. See e.g. the first callout under this YAML sample: https://57600--docspreview.netlify.app/openshift-enterprise/latest/machine_management/creating_machinesets/creating-machineset-vsphere.html#machineset-yaml-vsphere_creating-machineset-vsphere